apps/docs/components/Options.tsx

```diff
@@ -69,8 +69,8 @@ const Option: FC<IOption> = (props) => {
               REQUIRED
             </div>
           )}
-        </span>
-        <span className="text-foreground-muted text-xs">{props.type ?? 'no type'}</span>
+        </span>}
+        {!props.isEnum && <span className="text-foreground-muted text-xs">{props.type ?? 'no type'}</span>}
```

Comment:
I have to add a new prop to prevent require and type fields rendering. It looks weir for accepted enum values
<img width="608" alt="Screenshot 2024-05-15 at 9 49 35 AM" src="https://github.com/supabase/supabase/assets/689843/db9b4e32-3041-4999-b150-dd5715658e16">